app/tests/beaker-plugin-safe-app/test_modules/mutable_data_permissions_set.js

```diff
@@ -0,0 +1,34 @@
+let should = require('should');
+let testHelpers = require('./helpers');
+
+describe('window.safeMutableDataPermissionsSet', () => {
+  // QUESTION: Please review the following statement. Is it accurate?
+  it('sets allowable actions on a Mutable Data structure by an app, represented by the app\'s signing key', async () => {
```

Comment:
I'm not sure about the `apps signing key` as being representative... @krishnaIndia @bochaco does it make sense to yous?

Reply:
Yes, and the test is just missing the step to effectively insert that `PermissionsSet` into the MDs mapped to the app's sign key.